src/ActionsImporter.UnitTests/Services/ConfigurationServiceTests.cs

```diff
@@ -97,4 +98,28 @@ public async Task ReadCurrentVariablesAsync_FileExists_ReturnsVariables()
         // Assert
         Assert.AreEqual(expectedResult, result);
     }
+
+    [Test]
+    [Ignore("This test needs some sort of mock for Prompt")]
```

Comment:
@ethanis Curious if there's a good way you know of to mock the Prompt?

AFAICT, we'd have to make ConfigurationService have a new override that lets us pass in a mocked prompt. But that seems a bit contrived and too much of a change to make for a test.

Should I just not unit test this? Is there another way to go about mocking the input to the `Prompt` library?

Reply:
Unfortunately, I don't think there's a great way to mock the prompt. As you called out, we'd need to inject a service to mock and, at that point, the tests wouldn't be provided much coverage anyways.

I think it's ok to not add unit tests for this method 👍